src/components/cc-global-footer.vue

```diff
@@ -3,12 +3,8 @@
         <div class="container">
             <div class="columns">
                 <div class="column is-one-quarter">
-                    <a href="https://creativecommons.org" aria-label="home" class="main-logo margin-bottom-bigger">
-                        <span class="has-text-white">
-                            <svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid meet" viewBox="0 0 304 73">
-                                <use href="logos/cc/logomark.svg#logomark"></use>
-                            </svg>
-                        </span>
+                    <a href="/" aria-label="home" class="main-logo margin-bottom-bigger">
+                       <img src="../../assets/cc_logo_white.png" class="w-full" alt="Creative Commons Logo">
```

Comment:
Does this relative path work in the build output? We may need to figure out how to include the logo when publishing the package.

Reply:
The build command fails with this error: 
```
[!] Error: Unexpected character '�' (Note that you need plugins to import files that are not JavaScript)
assets/cc_logo_black.png (1:0)
1: �PNG
   ^
```

Reply:
One possible solution is to embed the binary encoded image directly in the template:

- https://stackoverflow.com/a/2429959

Reply:
So now we've gone from embedded SVG, which would ostensibly allow us to change the color via CSS, to two embedded PNG files. I'm not sure which trade-off is better, two embedded PNGs or figuring out how to keep the SVG from misbehaving.

Reply:
Base64 encoding the image will make the code very verbose

Reply:
Relatedly, here is the line to include the SVG logo in the main Vocabulary site:

```html
<div class="has-text-black">
    <style>svg { height: 73px; }</style>
    <svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid meet" viewBox="0 0 304 73">
        <use href="logos/cc/logomark.svg#logomark"></use>
    </svg>
</div>
```

Reply:
https://cc-vocabulary.netlify.app/?path=/story/assets-logos-main--logomark

Reply:
This SVG works well. I must have been using a broken SVG

Reply:
OK, great! Let's go with SVG haha.